scripts/git-pr.sh

```diff
@@ -66,9 +66,10 @@ echo "Pushing topic branch to origin..."
 git push -u origin $branch_name
 
 # Generate a Github pull request
-echo "Generating Github pull request..."
-pull_request_url="https://github.com/CarperAI/nmmo-environment/compare/$MASTER_BRANCH...CarperAI:nmmo-environment:$branch_name?expand=1"
-
-echo "Pull request URL: $pull_request_url"
-
+read -p "Do you like to create a PR to the CarperAI/nmmo-environment repo? (y/n) " ans
```

Comment:
Do we need to ask the user if they want to create a PR? All this does is generate a URL so there is no reason to bother the user with this question.

If we do want to ask the user, let's do it before pushing the branch?

Reply:
Oh, I thought it was actually making a PR and commented out. If it's just providing the url, I agree that there is no reason to bother. 

Thanks, David!